Training/Easy/chuck_norris.rb

```diff
@@ -27,25 +27,24 @@ def split_series(array)
     result
   end
 
-  def norris_parse(array)
-    result = ''
-    array.each do |a|
-      if a[0].zero?
-        result += '00 '
-      else
-        result += '0 '
-      end
-      result += '0' * a[1] + ' '
+  def norris_parse(series_array)
+    parsed = series_array.each_with_object('') do |a, result|
+      result << if a[0].zero?
```

Comment:
It's better to give meaning to the object's elements if it's not clear what they are - so `a[0]` and `a[1]` should have some meaning. You can assign them to local variables and then use them